packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -179,18 +194,25 @@ function TokensPanel({
 
   const tokensToShow = useMemo(() => {
     const tokenSearch = newToken.trim().toLowerCase()
+    const tokenAddresses = [
+      ...Object.keys(tokensFromUser),
+      ...Object.keys(tokensFromLists)
+    ]
+    if (!isDepositMode && isNetwork(l2Network.id).isArbitrumOne) {
+      tokenAddresses.push(CommonAddress.ArbitrumOne.USDC)
```

Comment:
NIT: Are we sure `CommonAddress.ArbitrumOne.USDC` will never be a part of any token-list? Reason I ask is because if it is, in the future, we will be pushing `CommonAddress.ArbitrumOne.USDC` twice in `tokenAddresses` object then. A fail-safe is to push USDC before we are de-duping the tokenAddress in line 197.

Reply:
no because it's unbridgeable by design